etl/steps/data/grapher/democracy/2023-03-02/vdem.meta.yml

```diff
@@ -0,0 +1,173 @@
+
+          <code>v2x_polyarchy = .5 ∗ MPI + .5 ∗ API</code>
+
+          <code>= .5 ∗ (v2x_elecoff ∗ v2xel_frefair ∗ v2x_frassoc_thick∗ v2x_suffr ∗ v2x_freexp_altinf) + .5 ∗ ((1/8) ∗ v2x_elecoff + (1/4) ∗ v2xel_frefair + (1/4) ∗ v2x_frassoc_thick + (1/8) ∗ v2x_suffr + (1/4) ∗ v2x_freexp_altinf)</code>
+
+          \[Text from [V-Dem Codebook v13](https://www.v-dem.net/documents/24/codebook_v13.pdf)\]
+        description_processing: |-
+          We ingested the original data from the [V-Dem project website](https://v-dem.net/data/the-v-dem-dataset/) and processed it with Stata.
+
+          We standardized the names of countries and regions by using the _[Our World in Data](https://ourworldindata.org/world-region-map-definitions)_ [standard entity names](https://ourworldindata.org/world-region-map-definitions).
+
+          We expanded the years and countries covered by V-Dem. To include more of the period when current countries were still non-sovereign territories, we identified the historical entity they were a part of and used that regime’s data whenever available.
+
+          For example, V-Dem only provides regime data since Bangladesh’s independence in 1971. There is, however, regime data for Pakistan and the colony of India, both of which the current territory of Bangladesh was a part. We, therefore, use the regime data of Pakistan for Bangladesh from 1947 to 1970, and the regime data of India from 1789 to 1946. We did so for all countries with a past or current population of more than one million.
+
+          We calculated regional and global averages of the Electoral Democracy Index and its sub-indices, weighted and unweighted by population.
+
+          All code and data is available [on GitHub](https://github.com/owid/notebooks/tree/main/BastianHerre/democracy).
+        presentation:
+          # variantSource
+          title_variant: V-Dem
+          # How does the producer --- describe this data?
+          attribution_short: V-Dem
+          topic_tags:
+            - Democracy
+          faqs:
+            - fragment_id: vdem-characteristics
+              gdoc_id: 1gGburArxglFdHXeTLotFW4TOOLoeRq5XW6UfAdKtaAw
+            - fragment_id: vdem-scoring
+              gdoc_id: 1gGburArxglFdHXeTLotFW4TOOLoeRq5XW6UfAdKtaAw
+            - fragment_id: vdem-coverage
+              gdoc_id: 1gGburArxglFdHXeTLotFW4TOOLoeRq5XW6UfAdKtaAw
+            - fragment_id: vdem-measured
+              gdoc_id: 1gGburArxglFdHXeTLotFW4TOOLoeRq5XW6UfAdKtaAw
+            - fragment_id: vdem-more-info
+              gdoc_id: 1gGburArxglFdHXeTLotFW4TOOLoeRq5XW6UfAdKtaAw
+          grapher_config:
```

Comment:
@danyx23 I used the **Copy as YAML** button from grapher and pasted grapher_config as it is to data pages. Can I please ask you to check a couple of these if any fields are obviously wrong or useless and don't belong to the data page YAML?

I'm unsure about fields like `slug`, `$schema` or `relatedQuestions`. If we find such fields we should exclude them from **Copy as YAML** and show linting error.